### ISSUE TITLE
State of different stateless models are currently handled differently 

### ISSUE THREAD

**Issue by t-ober** [ie3-institute/simona#302]
As chris noted states in for example chp and bm plants are handled differently. We should on a longer time scale harmonize that.

**Comment by staudtMarius** [comment]
@sebastian-peter Is this still the case with the new participant models?

**Comment by sebastian-peter** [comment]
Nope, thanks for the hint. `ChpModel` is gone and `BmModel` has been transformed into the new model structure.
